src/popover/popover.styles.scss

```diff
@@ -10,7 +10,7 @@
   opacity: 0;
   position: fixed;
   top: 0;
-  transition: all $default-transition;
+  transition: z-index $default-transition, opacity $default-transition;
```

Comment:
are you sure about this?

Reply:
I had a sass-lint warning on the usage of `all` on `transition`.
It is totally ok to define transition on multiple properties : https://developer.mozilla.org/fr/docs/Web/CSS/CSS_Transitions/Utiliser_transitions_CSS#Appliquer_une_transition_sur_plusieurs_propri%C3%A9t%C3%A9s

Reply:
This is ok, but not in this PR I suppose since you set it to "all" in this PR some time ago: https://github.com/alakarteio/kriya/pull/108

Reply:
Before `all` it was `opacity` but there was a problem with z-index.
Transition only on `z-index` do not produce the wanted effect.
I found that the combination of both produces the expected behaviour :smile:

Reply:
Yes why not, but why fixing this in this PR?
Is it because of a dependency update?

Reply:
yes updating sass-lint made this warning appear

Reply:
ok